blueprints/app/files/app/router.js

```diff
@@ -1,12 +1,14 @@
 import Ember from 'ember';
 import config from './config/environment';
 
-const Router = Ember.Router.extend({
+const { Router } = Ember;
```

Comment:
I'd prefer to have this be `const { Router: EmberRouter } = Ember;` and use `Router` for the apps router.